src/java.base/share/conf/security/java.security

```diff
@@ -1291,18 +1294,36 @@ jceks.key.serialFilter = java.base/java.lang.Enum;java.base/java.security.KeyRep
 #
 # The categories are:
 #
-#  hostInfo - IOExceptions thrown by java.net.Socket and the socket types in the
-#             java.nio.channels package will contain enhanced exception
-#             message information
+#  socket   - IOExceptions thrown by the java.net socket types (for example, Socket,
+#             ServerSocket, DatagramSocket) and the NetworkChannel types in the
+#             java.nio.channels package will contain enhanced exception message
+#             information
+#
+#  addressLookup - UnknownHostExceptions and other exceptions thrown by the
+#             java.net.InetAddress class
+#
+#  net      - All other exceptions thrown in networking code not included
+#             in the categories above. This include URL/URI, NetworkInterface
+#             and URLConnection among others.
+#
+#  hostInfo - Special value which signifies the three categories above combined
+#             (socket, addressLookup, net). This is provided for compatibility
+#             with previous releases.
```

Comment:
Are you sure "socket", "addressLookup", and "net" categories are useful and needed? I would think someone configuring this is looking to keeping sensitive fields out of the exception messages without knowing exactly which APIs are used.